[IMP] Inventory: Inter-warehouse doc update (#938)
odoo/documentation: inventory/routes/concepts/inter_warehouse.rst

=====================================

> Proceed to inter-warehouses transfers

Author: Honestly, I kept the warehouses because it was the name in the previous version.
But I agree that going straight to the point is far better!

Let's go for "Proceed to". And, when the next display is available, I'll change it again if it's better without the "proceed to" :)

> Proceed to inter-warehouses transfers

Reviewer: Well, @Lau-Odoo that's it.
We do have the new doc now hahaha

=====================================

When owning several warehouses, you might need to transfer goods from one warehouse to another.

> Proceeding to such behavior is called *inter-warehouses transfers*. If this action is physically

Author: Of course, if we change the warehouses for warehouse in the doc title, it must be done in the flow too :)

easy to perform, the administrative part can be arduous. Fortunately, Odoo comes with an intuitive

> flow helping save time when registering those transfers.

Author: I don't mind changing "helping" for "that helps." But then, that would be "that helps save time," and not "that helps to save time."

Configuration
=============

> First of all, go to :menuselection:`Configuration --> Settings` and enable **Storage Location**.

Reviewer 2: Storage Locations

Then, hit save.

.. image:: inter_warehouse/feature-storage-location.png
   :align: center

>    :alt: Image of the storage location feature being enabled.

Author: Hahaha, are you sure about that? 😆

>    :alt: Image of the storage location feature being enabled.

Reviewer: Well, you make me doubt myself, now...
Maybe it's an alt tag about a diplodocus. Who knows :s

>    :alt: Image of the storage location feature being enabled.

Author: You'll never know!

.. tip::

>    If you want to manage several routes within your warehouses, then tick the **Multi-Step Routes**

Author: 👍

   and check :doc:`use-routes`.

Creating a new warehouse
========================

The next step is to create your new warehouse. To do so, go to :menuselection:`Configuration -->

> Warehouse Management --> Warehouses`. Once there, click on **Create**.

Author: 👍

> Fill in a **Warehouse Name** and a **Short Name**. The short name is 5 characters maximum.

Reviewer 2: Jonathan told me: "We decided to write in American English, therefore we use 'fill out' instead of 'fill in'."

> Fill out a **Warehouse Name** and a **Short Name**. The short name is 5 characters maximum.

Author: 👍

.. image:: inter_warehouse/create-new-warehouse.png
   :align: center

>    :alt: View of a new warehouse being created.

Author: 👍

> .. tip::

Author: 👍

   Note that the **Short Name** is important as it appears on your transfer orders and other
   warehouse documents. It might be smart to use an understandable one (e.g.: WH/[first letters of

>    location]).

Author: 👍

Now, go back to your dashboard. There, new operations related to your warehouse have been
automatically generated.

.. image:: inter_warehouse/new-transfer-types.png
   :align: center
   :alt: Inventory app dashboard displaying new transfer type for the recently created warehouse.

Creating a new inventory
========================

If you create a new warehouse, you might already have an existing inventory in it. In that case, you
should create an inventory in Odoo. If this is not the case, you can skip this step.

Into the inventory application, select :menuselection:`Operations --> Inventory Adjustment`.
Then, create a new inventory by clicking on **Create**. Fill in the **Inventory Reference**,
**Date** and make sure to select the right warehouse and location.

.. image:: inter_warehouse/annual-inventory.png
   :align: center
   :alt: View of the inventory adjustment form before hitting the start button.

> Next, click on **Start Inventory**. A new window will open where you will be able to input your

Reviewer 2: You will be able to input your existing products in the new window.

existing products. Select add an item and indicate the **Real Quantity** available in the warehouse.
The theoretical quantity can not be changed as it represents a computed one from purchase and sales
orders.

.. image:: inter_warehouse/in-progress-annual-inventory.png
   :align: center
   :alt: View of the actual physical inventory of the new warehouse.

.. tip::

>    Don't forget to validate your inventory once you have recorder the state of all your products.

Reviewer 2: recorded

Create an internal transfer
===========================

The final step is to create your internal transfer. If you want to transfer 8 units of a product
from your first warehouse to another one, proceed as follows:

From your dashboard, select a internal movement of one of the two warehouses. To do so, click on

> :menuselection:`More  --> Planned Transfer`.

Author: I don't know, tbh. When you hover over the button, the text "Manage" appears. But I wrote more because when you click on it, more options appear 🙈

> :menuselection:`More  --> Planned Transfer`.

Reviewer: Well, that makes sense :D

.. image:: inter_warehouse/internal-transfer-choice.png
   :align: center
   :alt: View of the choice between planned transfer and immediate transfer.

> On the new window, select the source location zone (in this case your "old warehouse") and the
> destination location zone (in this case your "new" warehouse).

Reviewer: ```suggestion
On the new window, select the source location zone (in this case, your "old" warehouse) and the
destination location zone (in this case, your "new" warehouse).
```

> Add the products you want to transfer by clicking on **Add an Item** and don't forget to
> **Save** and **Mark as TODO** once you are done.

Reviewer: ```suggestion
Add the products you want to transfer by clicking on **Add an Item**, then click on **Save** and 
**Mark as TODO** once you are done.
```

.. image:: inter_warehouse/planned-internal-transfer.png
   :align: center
   :alt: View of the internal transfer form.

> If you selected **Immediate Transfer**, Odoo would have processed the transfer automatically after
> your click on **Validate**.

Reviewer: grammar: I believe the tenses don't follow well.
- If you selected --> Odoo _[simple present]_ processes the transfer
- - If you _[past perfect]_ had selected --> Odoo would have processed

```suggestion
If you selected **Immediate Transfer**, Odoo processes the transfer automatically after you clicked 
on **Validate**.
```

Author: Yup, I should have reread myself one more time x)

> your click on **Validate**.

Reviewer 2: you

If you select **Mark as TODO**, Odoo puts the transfer in **Waiting Availability** status.

> Click on **Reserve** to reserve the amount of products in your source warehouse.

Reviewer: ```suggestion
Click on **Reserve** to reserve the number of products in your source warehouse.
```

It is also possible to manually transfer each product:

- Via your dashboard, select the transfer order in the source location.

.. image:: inter_warehouse/in-progress-internal-transfer.png
   :align: center
   :alt: View of the dashboard with a zoom of the pending internal transfer.

- Select the right transfer order.

.. image:: inter_warehouse/internal-transfers-list.png
   :align: center
   :alt: View of the pending internal transfers list.

> - Click on the little pencil logo in the lower right corner in order to open the operation details
>   window. In this new window you can manually indicate how many products you process.

Reviewer: ```suggestion
- Click on the little pencil logo in the lower right corner to open the operation details window.
  In this new window, you can manually indicate how many products you process.
```

>   window. In this new window you can manually indicate how much products you process.

Reviewer 2: how many

.. image:: inter_warehouse/backorder.png
   :align: center
   :alt: View of a backorder pop-up window because all products are not moved at the same time.

> .. tip::

Reviewer: This is not a `tip` but a `note`
```suggestion
.. note::
```

>    If you decide to partially process the transfer order (e.g. a part of the products can't be
>    shipped yet due to an unexpected event), Odoo automatically asks if you wish to create a
>    **backorder**. Create a backorder if you expect to process the remaining products later,
>    do not create a backorder if you will not supply/receive the remaining products.

Reviewer: - put "partially" at the end
- e.g.**,** (comma) 

```suggestion
   If you decide to process the transfer order partially (e.g., a part of the products can't be
   shipped yet due to an unexpected event), Odoo automatically asks if you wish to create a
   **backorder**.

   - Create a backorder if you expect to process the remaining products later.
   - Do not create a backorder if you will not supply/receive the remaining products.
```
